.github/workflows/install_deps/action.yml

```diff
@@ -0,0 +1,113 @@
+name: Install Dependencies
+description: |
+  This workflow installs Python, PennyLane and all its dependencies. If a
+  requirements file is provided, it will upload the results of pip freeze
+  as well.
+inputs:
+  python_version:
+    description: The version of Python to use in order to run unit tests
+    required: false
+    default: '3.9'
```

Comment:
in case one day it isn't? idk. just keeping what we have on master:
https://github.com/PennyLaneAI/pennylane/blob/95129a0d6365b48cb4acfa828ceb6a8532e47ef5/.github/workflows/unit-test.yml#L26

Reply:
Fair enough. I'm usually of the opinion that less is better but I don't really mind in this case.